tox.ini

```
@@ -43,13 +43,12 @@ basepython = python2.7
 deps = -r{toxinidir}/test-requirements.txt
 
 [flake8]
-# E121: continuation line under-indented for hanging indent
-# E123: closing bracket does not match indentation of opening bracket’s line
 # E126: continuation line over-indented for hanging indent
 # E226: missing whitespace around arithmetic operator
 # W503: line break before binary operator
 # W504: line break after binary operator
-ignore=E121,E123,E126,E226,W503,W504
+ignore=E126,E226,E241,E402,E741,W503,W504
+ignore=E126,E226,W503,W504
```

Oops, I messed up resolving the merge conflicts:

```suggestion
ignore=E126,E226,W503,W504
```